src/Php84/bootstrap.php

```diff
@@ -60,3 +60,7 @@ function mb_ltrim(string $string, ?string $characters = null, ?string $encoding
         function mb_rtrim(string $string, ?string $characters = null, ?string $encoding = null): string { return p\Php84::mb_rtrim($string, $characters, $encoding); }
     }
 }
+
+if (function_exists('bcdiv')) {
+    function bcdivmod(string $num1, string $num2, ?int $scale = null): array {return p\Php84::bcdivmod($num1, $num2, $scale); }
```

Comment:
such polyfill function should be defined only when the bcmath extension is loaded (similar to how we deal with mbstring new functions just above)